airflow/jobs/scheduler_job.py

```diff
@@ -1374,6 +1363,7 @@ def _run_scheduler_loop(self) -> None:
         if not self.processor_agent:
             raise ValueError("Processor agent is not started.")
         is_unit_test: bool = conf.getboolean('core', 'unit_test_mode')
+        stale_dag_cleanup_timeout: int = conf.getint('scheduler', 'stale_dag_cleanup_timeout', 600)
```

Comment:
```suggestion
        stale_dag_cleanup_timeout = timedelta(seconds=conf.getint('scheduler', 'stale_dag_cleanup_timeout', 600))
```

Lets not re-create the timedelta object more often than we need to.